hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2090,45 +2090,6 @@ func (r *HostedClusterReconciler) reconcileControlPlaneOperator(ctx context.Cont
 		return fmt.Errorf("failed to reconcile controlplane operator rolebinding: %w", err)
 	}
 
-	// TODO: Remove this block after initial merge of this feature. It is not needed for latest CPO version
```

Comment:
Do you know what feature this refers to or if not im  just curious as to why were removing it?

Reply:
I do not. I know Seth removed most of the operator ingress stuff in another PR previously.

Reply:
https://github.com/openshift/hypershift/pull/1398 
At some point we were creating an ingress controller per hosted control plane, instead of us deploying our own router.